internal/app/cli/common.go

```diff
@@ -28,15 +38,37 @@ func printErrs(hdr string, errs []*repos.RepoAccessError) {
 	}
 }
 
+func printJSON(js any) {
```

Comment:
If I do e.g. a `tmc list --format json ...`  angle brackets are encoded for the repo.
```
{
    "name": "omnicorp/omnicorp/senseall",
    "author": "omnicorp",
    "manufacturer": "omnicorp",
    "mpn": "senseall",
    "repo": "\u003c.local/a2\u003e"
  },

```
I think the encoding is meant for browsers not to interpret it as HTML.
As we print it on stdout, maybe we could/should prevent this encodung for HTML.

Reply:
fixed